python/tk_multi_loader/delegate_publish_history.py

```diff
@@ -95,18 +213,9 @@ def _on_before_paint(self, widget, model_index, style_options):
         except:
             pass
             
-            
         # set the little description bit next to the artist icon
-        if sg_item.get("description") is None:
-            desc_str = "No Description Given"
-        else:
-            desc_str = sg_item.get("description")
-        
-        if sg_item.get("created_by") is None:
-            author_str = "Unspecified User"
-        else:
-            author_str = "%s" % sg_item.get("created_by").get("name")
-
+        desc_str = sg_item.get("description") or "No Description Given"         
+        author_str = sg_item.get("created_by").get("name") or "Unspecified User"
```

Comment:
actually, I recently had a bug where a field from a shotgun query unexpectedly returned None instead of a dictionary - is it possible in this case that created_by may be None?  In which case splitting across multiple statements and checking each obtained value would be safer...